api/migrations/versions/b3f7a73c98f3_.py

```diff
@@ -57,3 +65,311 @@ def downgrade():
     op.drop_table('application_form')
     # ### end Alembic commands ###
 
+def seed_application_form_data(application_form):
+    op.bulk_insert(application_form,
+    [
+        {
+            'id': 1,
+            'event_id': 1,
+            'is_open': True,
+            'deadline': '30 April 2019'
+        }
+    ])
+
+def seed_section_data(section):
+    op.bulk_insert(section,
+    [
+        {
+            'id': 1,
+            'application_form_id': 1,
+            'name': 'Indaba 2019 Application Form',
+            'description': 'This is the official application form to apply for participation in the Deep Learning Indaba to be held 25-31 August 2019 in Nairobi, Kenya. Students can also use this application form to apply for travel and accommodation awards. \n \n \n Closing date: 30 April 2019',
+            'order': 1            
+        },
+        {
+            'id': 2,
+            'application_form_id': 1,
+            'name': 'Tell Us a Bit About You',
+            'description': 'Please use this section to tell us a bit more about yourself and your intentions as a future Deep Learning Indaba ambassador.',
+            'order': 2            
+        },
+        {
+            'id': 3,
+            'application_form_id': 1,
+            'name': 'Apply For Travel Support ',
+            'description': 'We may be able to sponsor the cost of travel and accommodation for some attendees. These travel awards are limited and highly competitive, but are assessed independently of Indaba attendance: applying for a travel award neither enhances nor undermines your chances of being accepted. To help as many people attend as possible, before applying for travel support, please check if your supervisor, department or university is able to support you in any way. ',
+            'order': 3            
+        },
+        {
+            'id': 4,
+            'application_form_id': 1,
+            'name': 'Demographic Information',
+            'description': 'We regard the promotion of diversity, inclusivity and transformation as a high priority. The information below helps us track our progress in this regard.',
+            'order': 4            
+        },
+        {
+            'id': 5,
+            'application_form_id': 1,
+            'name': 'Previous Attendance',
+            'description': 'Help us quantify our progress by giving us some info on your previous Indaba experiences.',
+            'order': 5            
+        },
+        {
+            'id': 6,
+            'application_form_id': 1,
+            'name': 'Additional Information',
+            'description': 'Anything else you may want us to know',
+            'order': 6           
+        }
+    ])
+
+def seed_question_data(question):
+    op.bulk_insert(question,
+    [
+        {
+            'id': 1,
+            'application_form_id': 1,
+            'section_id': 1,
+            'type': 'short-text',
+            'headline': 'Email address',
```

Comment:
We can remove this question because we already have their email address from their user profile :)

Reply:
Big oof. Got it